1-CONTRIBUTION-GUIDE/README.md

```diff
@@ -1,140 +1,106 @@
-9. Name **variables** using this scheme **templateScenarioResourceName** (e.g. simpleLinuxVMVNET, userRoutesNSG, elasticsearchPublicIP etc.) that describe the scenario rather. This ensures when a user browses all the resources in the Portal there aren't a bunch of resources with the same name (e.g. myVNET, myPublicIP, myNSG)
-10. **Storage account names** need to be lower case and can't contain hyphens (-) in addition to other domain name restrictions. These also need to be globally unique.
-11. **Passwords** must be passed into parameters of type `securestring`.
-    * Passwords must also be passed to customScriptExtension using the `commandToExecute` property in `protectedSettings`. This will look like below
-
-    ```
-     "properties": {
-       "publisher": "Microsoft.OSTCExtensions",
-       "type": "CustomScriptForLinux",
-       "typeHandlerVersion": "1.4",
-       "settings": {
-       "fileUris": [
-     	   "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/lamp-app/install_lamp.sh"
-       ]
-     },
-     "protectedSettings": {
-       "commandToExecute": "[concat('sh install_lamp.sh ', parameters('mySqlPassword'))]"
-       }
-     }
-    ```
-
-12. Every parameter in the template must have the **lower-case description** tag specified using the metadata property. This looks like below
-
-  ```json
-  "newStorageAccountName": {
-        "type": "string",
-        "metadata": {
-            "description": "The name of the new storage account created to store the VMs disks"
-        }
-  }
-  ```
+## Files, folders and naming conventions
 
+1. Every deployment template and its associated files must be contained in its own **folder**. Name this folder something that describes what your template does. Usually this naming pattern looks like **appName-osName** or **level-platformCapability** (e.g. 101-vm-user-image) 
+ + **Required** � Numbering should start at 101. 100 is reserved for things that need to be at the top.
+ + **Protip** - Try to keep the name of your template folder short so that it fits inside the Github folder name column width.
+2. Github uses ASCII for ordering files and folder. For consistent ordering **create all files and folders in lowercase**. The only **exception** to this guideline is the **README.md**, that should be in the format **UPPERCASE.lowercase**.
+3. Include a **README.md** file that explains how the template works. 
+ + Guidelines on the README.md file below.
+4. The deployment template file must be named **azuredeploy.json**.
+5. There should be a parameters file named **azuredeploy.parameters.json**. 
+ + Please fill out the values for the parameters according to rules defined in the template (allowed values etc.), For parameters without rules, a simple "changeme" will do as the acomghbot only checks for syntactic correctness using the ARM Validate Template [API](https://msdn.microsoft.com/en-us/library/azure/dn790547.aspx).
```

Comment:
let's change this since we now need placeholder parameters. Let's link that section here.